src_testbed/testbed.rs

```diff
@@ -864,13 +867,15 @@ fn update_testbed(
     #[cfg(feature = "other-backends")] mut other_backends: NonSendMut<OtherBackends>,
     mut plugins: NonSendMut<Plugins>,
     ui_context: Res<EguiContext>,
-    mut gfx_components: Query<(&mut Transform,)>,
+    gfx_components: Query<(&mut Transform,)>,
     mut cameras: Query<(&Camera, &GlobalTransform, &mut OrbitCamera)>,
 ) {
     let meshes = &mut *meshes;
     let materials = &mut *materials;
     let prev_example = state.selected_example;
 
+    let gfx_components = Arc::new(Mutex::new(gfx_components));
+
```

Comment:
I'm not sure if Arc<Mutex<>> is the way to go here, but passing the mutable transform is not possible without it. Not sure yet if the gfx_components will need to be a mutable query yet..

Reply:
This is odd. I tried removing your `Arc<Mutex<>>` and it compiled fine. Did I miss something?